en/08_Changelogs/6.0.0.md

```diff
@@ -857,7 +876,7 @@ SilverStripe\ORM\Connect\MySQLDatabase:
 
 ### Run `CanonicalURLMiddleware` in all environments by default {#url-middleware}
 
-In Silverstripe CMS 5 [`CanonicalURLMiddleware`](api:SilverStripe\Control\Middleware\CanonicalURLMiddleware) only runs in production by default. This lead to issues with `fetch` and APIs behaving differently in production environments to development. Silverstripe 6.0 changes this default to run the rules in `dev`, `test`, and `live` by default.
+In Silverstripe CMS 5 [`CanonicalURLMiddleware`](api:SilverStripe\Control\Middleware\CanonicalURLMiddleware) only runs in production by default. This lead to issues with some functionality behaving differently in production environments compared to development. Silverstripe 6.0 changes this default to run the rules in `dev`, `test`, and `live` by default.
```

Comment:
I _presume_ `fetch` was referring to the javascript function.... but really it doesn't matter what specifically was behaving differently.